source/future.tex

```diff
@@ -1468,8 +1468,7 @@
 only if the type \tcode{T} is any of the following:
 \begin{itemize}
 \item a function type or a pointer to function type taking one argument of type \tcode{T1}
-% FIXME: Should this be R T0::f() ?
-\item a pointer to member function \tcode{R T0::f} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
+\item a pointer to member function \tcode{R T0::f()} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
```

Comment:
This is talking about a type `T`. There's no way a type can be "a pointer to member function `size_t string::size() const`" -- that's not a property of a type.

My concerns here are twofold:
1) This may be a normative change, depending on whether an `argument_type` was also supposed to be defined for a pointer to member function with parameters or not. (With your change we seem to say only the no-parameter case is covered, whereas without it, this appears to apply regardless of whether the function has parameters.)
2) This change still leaves the wording meaningless.

I think we do want to make a change in this direction (and I suspect the right change would be to use `R T0::*f()`*`cv`*), but I'd like LWG to look at this and decide what exactly to do.